anyway/infographics_utils.py

```diff
@@ -407,6 +414,10 @@ def create_infographics_data(news_flash_id, number_of_years_ago):
     start_time = datetime.date(
         end_time.year - number_of_years_ago, 1, 1)
 
+    last_accident_date=get_latest_accident_date(table_obj=AccidentMarkerView, filters=[], start_time=start_time, end_time=end_time)
```

Comment:
@galrou I suggest sending None in filters instead of [] - I think it might be better practice, especially when the first line in the function get_latest_accident_date is `filters= filters or {}` (even though you get the wanted result either way)